stats.js

```diff
@@ -84,14 +89,15 @@ function buildStats(data) {
       const eloChange = EloChange(player.eloHistory.length);
       let diff;
       if (winner === "imposter") {
-        diff = eloChange(player.elo, crewAvgElo)[0];
+        diff = eloChange(player.imposterElo, crewAvgElo)[0];
         player.imposterWin += 1;
-        player.elo += diff;
+        player.imposterElo += diff;
       } else {
-        diff = eloChange(player.elo, crewAvgElo)[1];
+        diff = eloChange(player.imposterElo, crewAvgElo)[1];
         player.imposterLoss += 1;
-        player.elo += diff;
+        player.imposterElo += diff;
       }
+      player.elo = (player.crewElo + player.imposterElo) / 2
```

Comment:
put a semicolon you fucking animal

Reply:
4/5 and 1/5 weighting perhaps? Due to 20% chance of being imposter in any game 😇